Expecto/Expecto.fs

```diff
@@ -842,10 +853,18 @@ module Impl =
 
         exn = fun n e d -> async {
           do! innerPrinter.beforeEach n
+          let rec msgWithInner (ex: exn) msg =
+            let currentMsg = 
+              msg + (sprintf "%s%s" (ex.Message) Environment.NewLine)
+            if isNull ex.InnerException then
+              currentMsg
+            else
+              msgWithInner ex.InnerException currentMsg
+
           tcLog "testFailed" [
             "flowId", formatName n
             "name", formatName n
-            "message", e.Message
```

Comment:
Have you verified this works on TeamCity? Otherwise, I think we should just not change this location in this PR.

Reply:
I couldn't do that for now, so I will remove changes from here for now :(